docs/spec/trustbloc-did-method.md

```diff
@@ -6,7 +6,7 @@ This is version `0.1` of the TrustBloc DID Method Specification.
 ## Introduction
 _This section is non-normative_
 
-The `did:trustbloc` DID method allows groups of independent entities to share custody of a DID registry consisting of [Sidetree](https://identity.foundation/sidetree/spec/) over a permissioned ledger.
+The `did:trustbloc` DID method allows groups of independent entities to share custody of a DID registry consisting of [Sidetree](https://identity.foundation/sidetree/docs/spec/) over a permissioned ledger.
```

Comment:
We should be more explicit that this link is to the the Sidetree protocol spec.


Perhaps an additional sentence like:

> For more information on the Sidetree protocol, please refer to the [Sidetree protocol specification](https://identity.foundation/sidetree/docs/spec/).

Reply:
Fixed in https://github.com/trustbloc/trustbloc-did-method/pull/70